qa/tasks/cephfs/test_data_scan.py

```diff
@@ -458,7 +458,7 @@ def validate(self):
 class TestDataScan(CephFSTestCase):
     MDSS_REQUIRED = 2
 
-    def _rebuild_metadata(self, workload, workers=1, unmount=True):
+    def _rebuild_metadata(self, workload, workers=1, unmount=True, parallelize_cleanup=True):
```

Comment:
Is this really required? Won't using `worker_count=workers` suffice?

Reply:
Better to remove this IMO.

Reply:
done, PTAL @vshankar